_projects/2018/poliastro/export-czml.md

```diff
@@ -0,0 +1,72 @@
+---
+name: Export to CZML
+desc: Implement export of Orbit objects to CZML to visualize in Cesium.js
+requirements:
+ - Familiar with the JSON format
+ - Basic experience with JavaScript or TypeScript
+difficulty: medium
+issues:
+ - https://github.com/poliastro/poliastro/issues/117
+ - https://github.com/poliastro/poliastro/issues/287
+mentors:
+ - juanlu001
+initiatives:
+ - GSOC
+ - SOCIS
+tags:
+ - python
+ - javascript
+collaborating_projects:
+ - poliastro
+---
+
+#### Description
+
+Since 2017 poliastro features 3D plotting using Plotly, as can be seen in this
+notebook:
+
+http://docs.poliastro.space/en/latest/examples/Plotting%20in%203D.html
```

Comment:
Instead of saying `as can be seen in this notebook: <link>` could you say: `[poliastro features 3d plotting using Plotly](<link>)`.

Reply:
Don